netci.groovy

```diff
@@ -535,6 +537,10 @@ def static getJobName(def configuration, def architecture, def os, def scenario,
             if (os.toLowerCase() == "windows_nt") {
                 // These are cross builds
                 baseName = architecture.toLowerCase() + '_cross_' + configuration.toLowerCase() + '_' + os.toLowerCase()
+
+                if (configuration == 'pri1') {
```

Comment:
This is not necessary: (1) 'pri1' is a scenario, not a configuration, and (2) because of that, "_pri1" is already being added to the name due to the 'suffix' definition. Same for the cases below.